src/index.js

```diff
@@ -68,6 +68,7 @@ export default class AuthnWidget {
     this.invokeReCaptcha = options && options.invokeReCaptcha;
     this.checkRecaptcha = options && options.checkRecaptcha;
     this.grecaptcha = options && options.grecaptcha;
+    this.scriptLocation = options && options.scriptLocation;
```

Comment:
How reusable is this script?  I think it might be good to either have this be an extensible map or name it to be a little more specific to its current use. It could get confusing in the future if we need more script locations for different purposes.

```scriptsLocation = { IDW: "path-to-script", THX-WEB: "path-to-script" }```

Reply:
Changed to deviceProfileScript for now the kit only support two type of kits that needs device profiling (and IDW don't need a local script).